#!/usr/bin/env python
PACKAGE = "project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#gen.add("odometry_integration_type", int_t, 0, "0 euler, 1 runge-kutta", 0, 0, 1)

size_enum = gen.enum([gen.const("euler", int_t, 0, "Euler integration"),
					  gen.const("rk", int_t, 1, "Runge-Kutta integration")], "enum to set the odometry integration type")

gen.add("odometry_integration_type", int_t, 2, "the integration type parameter which is edited via an enum", 0, 0, 1, edit_method=size_enum)

exit(gen.generate(PACKAGE, "project1", "dynamic_rec"))